src/openforms/registrations/contrib/email/plugin.py

```diff
@@ -46,6 +49,29 @@ def register_submission(self, submission: Submission, options: dict) -> None:
 
         attachments = submission.attachments.as_mail_tuples()
 
+        attachment_formats = options.get("attachment_formats", [])
+        extra_attachments = []
+        for attachment_format in attachment_formats:
+            mime_type = types_map[f".{attachment_format}"]
+            if attachment_format in [AttachmentFormat.csv, AttachmentFormat.xlsx]:
+                export_data = create_submission_export(
+                    Submission.objects.filter(pk=submission.pk)
+                ).export(attachment_format)
+
+                attachment = (
+                    f"{submission.form.name} - submission.{attachment_format}",
+                    export_data,
+                    mime_type,
+                )
+            elif attachment_format == AttachmentFormat.pdf:
+                attachment = (
+                    submission.report.title,
+                    submission.report.content.read(),
+                    mime_type,
+                )
```

Comment:
@joeribekker the submission pdf was not yet added as an attachment, so I added this here